### ISSUE TITLE
Problem about the Pretrained Model

### ISSUE THREAD

**Issue by JiYuanFeng** [CSAILVision/semantic-segmentation-pytorch#85]
Hi！
I try to use the pretrained model(resnet101-imagenet.pth) as the encoder module in my network, 
I use my own project code and datasets.
And when training the model ,the output（loss,score）seems right, But after a training epoch ,i use the model to eval, set model.eval() ,And there is nothing in the model output, 
 ![image](https://user-images.githubusercontent.com/20653249/41806647-6b50ac4c-76f4-11e8-9085-30c878053238.png)
And the below is  train outputs and val outputs
![image](https://user-images.githubusercontent.com/20653249/41806658-beba3c9a-76f4-11e8-9fe0-864563dd9a3e.png)

And i have set the model to eval mode when doing the evaluation,
It's there any potential mistakes in the my using method?

**Comment by Tete-Xiao** [comment]
Sorry there's nothing we can do due to the limited information.
